src/Workspaces/CoreTestUtilities/Workspaces/TestWorkspace_ILspWorkspace.cs

```diff
@@ -8,9 +8,9 @@
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+namespace Microsoft.CodeAnalysis.Test.Utilities
 {
-    public partial class TestWorkspace : Workspace, ILspWorkspace
```

Comment:
💭 This should still indicate directly that it implements `ILspWorkspace`, right?